### PR TITLE
Changed single-use berry blender strings to be COMPOUND_STRINGs

### LINKED ISSUE #4959: Convert single-use string variables to COMPOUND_STRINGs
### Description

From @Bassoonian in #4951 
>These are only used once, so I would personally be inclined to convert them to COMPOUND_STRINGs and remove them from strings.c and strings.h. The only reason they're lumped with the rest of the strings is for pret matching purposes and it just makes it harder to edit imo

I think this logic should be applied to all single-use strings

- [x] Berry Blender
- [x] PokeNav
- [x] Default player names
- [x] Mechadolls
- [x] PC text
- [x] Bag pockets
- [x] Berries
- [x] PokeBlocks

### Discord contact info
Frankfurter0

### PR DESCRIPTION
**Pull request by fdeblasio**
## Description
As part of the effort to turn single-use strings into COMPOUND_STRINGs, the berry blender strings were converted. Also removed unused berry blender text

## Issue(s) that this PR fixes
Partially fixes #4959 

## **Discord contact info**
Frankfurter0

### CODE REVIEW

**Comment by AlexOn1ine** [comment]
I've never learned it even though we started using them long ago but what is the advantage of using compound strings?

**Comment by fdeblasio** [comment]
> I've never learned it even though we started using them long ago but what is the advantage of using compound strings? Not a review. Just a general question.

There's definitely people who know more/can word things better then I can, but I think it's just to simplify things. A lot variables that are strings are defined in strings.h and set in strings.c and then called in various files. These variables are u8 arrays. If the variables are only used once, it's kinda a pain to have that variable in three different files. The COMPOUND_STRING macro casts a string as a u8 array, which eliminates the need to make a new variable for it.

This PR probably isn't the best example since all the string variables here are both defined and used in berry_blender.c, but the first commit with the NPC names kinda shows what I was going for.

File: src/berry_blender.c [hunk 1]
```diff
@@ -261,50 +261,26 @@ static const u8 sUnusedText_2[] = _("▶");
...
 
 static const u8 *const sBlenderOpponentsNames[] =
 {
-    [BLENDER_MISTER] = sText_Mister,
-    [BLENDER_LADDIE] = sText_Laddie,
-    [BLENDER_LASSIE] = sText_Lassie,
-    [BLENDER_MASTER] = sText_Master,
-    [BLENDER_DUDE]   = sText_Dude,
-    [BLENDER_MISS]   = sText_Miss
+    [BLENDER_MISTER] = COMPOUND_STRING("MISTER"),
```
Comment (AlexOn1ine) [inline src/berry_blender.c:271]: So I understand why this makes sense to be a compound string.

File: src/berry_blender.c [hunk 2]
```diff
@@ -1115,7 +1091,7 @@ static void CB2_LoadBerryBlender(void)
             sBerryBlender->mainState++;
         break;
     case 4:
-        if (PrintMessage(&sBerryBlender->textState, sText_BerryBlenderStart, GetPlayerTextSpeedDelay()))
+        if (PrintMessage(&sBerryBlender->textState, COMPOUND_STRING("Starting up the BERRY BLENDER.\pPlease select a BERRY from your BAG\nto put in the BERRY BLENDER.\p"), GetPlayerTextSpeedDelay()))
```
Comment (AlexOn1ine) [inline src/berry_blender.c:1094]: But isn't this to much? By this logic we would have to turn everything into a compound string and for some reason I don't feel like it is a good idea. (follow up to a previous comment.)

Comment (fdeblasio) [reply src/berry_blender.c:1094]: Yeah even when I was doing this commit, I felt like it was overkill. But I figured it wouldn't hurt to do the commit to get feedback and have to revert it

**Review by AlexOn1ine — APPROVED** [review]

**Review by DizzyEggg — APPROVED** [review]

**Pull request merged**